src/runners/rfqm_worker_runner.ts

```diff
@@ -89,21 +98,23 @@ export async function runRfqmWorkerAsync(rfqmService: RfqmService, workerAddress
         sqsClient,
         beforeHandle: async () => rfqmService.workerBeforeLogicAsync(workerAddress),
         handleMessage: async (message) => {
-            RFQM_JOB_DEQUEUED.inc();
+            RFQM_JOB_DEQUEUED.labels(workerAddress).inc();
             const { orderHash } = JSON.parse(message.Body!);
             logger.info({ workerAddress, orderHash }, 'about to process job');
-            return rfqmService.processRfqmJobAsync(orderHash, workerAddress);
+            const stopTimer = RFQM_JOB_PROCESSING_TIME.labels(workerAddress).startTimer();
+            rfqmService.processRfqmJobAsync(orderHash, workerAddress);
+            stopTimer();
```

Comment:
can we keep the `return` types the same? Forget if there's something that depends on the return type at all - most likely not, but just want to keep it consistent

Reply:
definitely! good spot!